EIPS/eip-2935.md

```diff
@@ -108,7 +111,7 @@ TBD
 
 ## Security Considerations
 
-Adding ~2.5 million storage slots per year bloats the state somewhat but not much relative to the hundreds of millions of existing state objects.
+Having contracts (system or otherwise) with hot update paths (branches) poses a risk of "branch" poisioning attracks where one could sprinkle trivial amounts of eth around these hot paths (branches). But it has been deemed that cost of attack would escalate significantly to cause any meaningful slow down of state root updates.
```

Comment:
```suggestion
Having contracts (system or otherwise) with hot update paths (branches) poses a risk of "branch" poisioning attacks where one could sprinkle trivial amounts of eth around these hot paths (branches). But it has been deemed that cost of attack would escalate significantly to cause any meaningful slow down of state root updates.
```